src/modules/auth/auth.module.ts

```diff
@@ -68,6 +71,16 @@ export class AuthModule implements NestModule {
         { path: '/v1/login/siwe/initiate', method: RequestMethod.POST },
         { path: '/v1/login/siwe/verify', method: RequestMethod.POST }
       )
-      .forRoutes({ path: '/*', method: RequestMethod.ALL });
+      .forRoutes({ path: '/*', method: RequestMethod.ALL })
+      .apply((_req: Request, res: Response, next: NextFunction) => {
+        if (!this.configService.get('BLOCKNUM_AUTH_ENABLED')) {
+          res
+            .status(404)
+            .send(
+              'Authentication at this endpoint is disabled. Other authentication protocols may be available'
```

Comment:
I think returning an object as a response would be more ideomatic
This would better match other 404 responses. See, for example, this attempt to `GET` to `login` endpoint:
```
❯ curl -X GET https://identitycache.energyweb.org/v1/login          
{"statusCode":404,"message":"Cannot GET /v1/login","error":"Not Found"}
```
```suggestion
{
    code: 404,
    error: 'Not Found',
    message: 'Authentication at this endpoint is disabled. Other authentication protocols may be available'
}    
```

Reply:
In my opinion status code and message serves different pusposes, so they should not be mixed. But I guess it will not hurt in include status in message
By the way you can get status by curl with `-i` option https://curl.se/docs/manpage.html#-i